lnbits/static/js/base.js

```diff
@@ -31,11 +31,17 @@ window.LNbits = {
         fiat_provider: fiatProvider
       })
     },
-    payInvoice(wallet, bolt11) {
-      return this.request('post', '/api/v1/payments', wallet.adminkey, {
+    payInvoice(wallet, bolt11, internalMemo = null) {
+      const data = {
         out: true,
         bolt11: bolt11
-      })
+      }
+      if (internalMemo && internalMemo !== '') {
```

Comment:
empty string is falsly: https://developer.mozilla.org/en-US/docs/Glossary/Falsy
so use of `if (internalMemo){` is sufficient